src/System.Management.Automation/engine/Modules/ModuleCmdletBase.cs

```diff
@@ -4848,6 +4853,21 @@ internal void CleanupWindowsPowerShellCompatResources()
             }
         }
 
+        internal static void SyncCurrentLocationHandler(object sender, LocationChangedEventArgs args)
+        {
+            PSSession compatSession = GetWindowsPowerShellCompatRemotingSession();
+            if (compatSession?.Runspace.RunspaceStateInfo.State == RunspaceState.Opened)
+            {
+                using var ps = System.Management.Automation.PowerShell.Create(RunspaceMode.CurrentRunspace);
```

Comment:
I do not think you need S.M.A here. 
```suggestion
                using var ps = PowerShell.Create(RunspaceMode.CurrentRunspace);
```

Reply:
Getting this without it:
` The type or namespace name 'Create' does not exist in the namespace 'Microsoft.PowerShell' (are you missing an assembly reference?)`